Gulpfile.js

```diff
@@ -13,7 +12,7 @@ var paths = {
 gulp.task("sass", function () {
   return gulp.src(paths.scss)
     .pipe(sass({
-      includePaths: ["styles"].concat(bourbon)
+        sourcemaps: true
```

Comment:
When I removed this line I began to encounter compilation errors. After googling it became apparent that sourcemaps was to blame for `gulp-sass` and `gulp-autocomplete` fighting each other. It seemed that these were supposed to be enabled, so I explicitly enabled them, and CSS began to flow again.